src/System.Net.Sockets/src/System/Net/Sockets/BufferList.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace System.Net.Sockets
```

Comment:
Saw the above comment: thanks for explaining.

As above: maybe renaming the inner type as well as ensuring that the file-name ends in .Linux.cs would avoid later confusion.